demo/admin/crud-generator-config.ts

```diff
@@ -8,4 +8,12 @@ export default [
         target: "src/news/generated",
         entityName: "News",
     },
+    {
+        target: "src/books/generated",
+        entityName: "Book",
+        rootBlocks: {
+            coverImage: { name: "DamImageBlock", import: "@comet/cms-admin" },
+            link: { name: "LinkBlock", import: "../../common/blocks/LinkBlock" },
```

Comment:
custom rootBlocks should not be neccasary for common cases and should be avoided

Reply:
Oh okay. I didn't get that. So it's not necessary for these two examples

Reply:
I added a notice to the config